content/docs/09-registries-and-packs/7-advanced-configuration.md

```diff
@@ -0,0 +1,211 @@
+---
+title: 'Advanced Registry configuration'
+metaTitle: 'Advanced Registry configuration'
+metaDescription: 'Customizing deployments of Spectro Registry'
+icon: ''
+hideToC: true
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+import Tooltip from "shared/components/ui/Tooltip";
+
+
+# Advanced Configuration of Spectro Registry
+
+You can modify a few deployment options of the Spectro Registy.
+Configuration is done by a YAML file, however you can override 
+many options using environment variables.
+
+The YAML file is divided into keys and values, which can have
+embedded keys and values, such as:
+
+```
+version: 0.1
+log:
+  level: info
+  fields:
+    environment: test
+```
+
+
+The key `version` has number value. The `log` key has value with multiple keys (`level`,
+`fields`), which in turn have more keys.
+
+To override the value of `log.level` you can specify an environment variable named
+`REGISTRY_LOG_LEVEL`.
```

Comment:
```suggestion
`REGISTRY_LOG_LEVEL`.

```shell
export REGISTRY_LOG_LEVEL=debug
```

```

Might be good to provide an example.